docs/paper/dev/api/command-api/arguments/registry-arguments.mdx

```diff
@@ -0,0 +1,270 @@
+import ResourceCatVariant from "./assets/registry-arguments/resource-cat-variant.mp4"
+import ResourceDamageType from "./assets/registry-arguments/resource-damage-type.mp4"
+import ResourceDataComponentType from "./assets/registry-arguments/resource-data-component-type.mp4"
+import ResourceFluid from "./assets/registry-arguments/resource-fluid.mp4"
+import ResourceFrogVariant from "./assets/registry-arguments/resource-frog-variant.mp4"
+import ResourceGameEvent from "./assets/registry-arguments/resource-game-event.mp4"
+import ResourceInstrument from "./assets/registry-arguments/resource-instrument.mp4"
+import ResourceItem from "./assets/registry-arguments/resource-item.mp4"
+import ResourceJukeboxSong from "./assets/registry-arguments/resource-jukebox-song.mp4"
+import ResourceMapDecorationType from "./assets/registry-arguments/resource-map-decoration-type.mp4"
+import ResourceMemoryModuleType from "./assets/registry-arguments/resource-memory-module-type.mp4"
+import ResourceMenu from "./assets/registry-arguments/resource-menu.mp4"
+import ResourceMobEffect from "./assets/registry-arguments/resource-mob-effect.mp4"
+import ResourcePaintingVariant from "./assets/registry-arguments/resource-painting-variant.mp4"
+import ResourceSoundEvent from "./assets/registry-arguments/resource-sound-event.mp4"
+import ResourceStructureType from "./assets/registry-arguments/resource-structure-type.mp4"
+import ResourceVillagerProfession from "./assets/registry-arguments/resource-villager-profession.mp4"
+import ResourceVillagerType from "./assets/registry-arguments/resource-villager-type.mp4"
+import ResourceWolfVariant from "./assets/registry-arguments/resource-wolf-variant.mp4"
+
+import ResourceKeyItem from "./assets/registry-arguments/resourcekey-item.mp4"
+import EnchantsRegistry from "./assets/registry-arguments/enchants-registry.mp4"
+
+# Registry Arguments
+Registries in Minecraft hold all sort of information - possible item materials, block materials, enchantments, potion effects, ... and more!
+
+There are two types of registry arguments: `resource` and `resourceKey`. 
+The main difference between those arguments is the return value: The `resource` argument returns the parsed value, whilst the `resourceKey` only returns a `TypedKey`, which
+you can use to retrieve the value yourself.
+
+## Resource Argument
+Just like any other argument, you can get a `ArgumentType<T>` reference to it using `ArgumentTypes.resource(RegistryKey<T>)`. A selection of possible registry keys can be
+found below. They are accessed in a static context using the <Javadoc name='io.papermc.paper.registry.RegistryKey'>`RegistryKey`</Javadoc> interface.
+
+Each entry in `RegistryKey` returns a `RegistryKey<T>`. The `<T>` generic parameter here describes the return type. This means that if we were to retrieve
+<Javadoc name='io.papermc.paper.registry.RegistryKey#ITEM'>`RegistryKeys.ITEM`</Javadoc>, the return type would be an `ItemType`, since it is defined as follows:

```

Comment:
```suggestion
<Javadoc name='io.papermc.paper.registry.RegistryKey#ITEM'>`RegistryKey.ITEM`</Javadoc>, the return type would be an `ItemType`, since it is defined as follows:
```